pkg/kubelet/container_manager_linux.go

```diff
@@ -141,10 +142,37 @@ func createManager(containerName string) *fs.Manager {
 	}
 }
 
+const sysctlVmOvercommitMemory = "vm/overcommit_memory"
+
+// disableKernelMemoryOvercommitHandling tells the kernel to perform no memory over-commit handling.
+// Under this setting, the potential for memory overload is increased, but so is performance for
+// memory-intensive tasks
+// sets /proc/sys/vm/overcommit_memory to 1
+func disableKernelMemoryOvercommitHandling() error {
```

Comment:
Is it possible to add some testing? Not necessary for this PR, but it might be useful, like for example conformance.